RecipeApi/Exceptions/ForbidException.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RecipeApi.Exceptions
+{
+    public class ForbidException: Exception
+	{
+		public ForbidException(string message) : base(message)
```

Comment:
You have a standard exception thrown from the fuction. It's 403 if I remember correctly.

Reply:
I create issue to delete all my exceptions and add standard exceptions